mlflow/cli.py

```diff
@@ -121,14 +121,16 @@ def run(uri, entry_point, version, param_list, experiment_id, mode, cluster_spec
                    "other machines.")
 @click.option("--port", "-p", default=5000,
               help="The port to listen on (default: 5000).")
-def ui(file_store, host, port):
+@click.option("--static-prefix", default=None,
+              help="A prefix which will be prepended to the path of all static paths.")
```

Comment:
I don't think we want this option on `mlflow ui` necessarily. Just put it on `mlflow server`. The `ui` command will be for people who want to run the UI locally and I'd actually like to make it open a web browser too eventually, so we should use `server` for stuff people want to host behind a proxy and such.

Reply:
Also, I would make the option a bit more descriptive: "A URL prefix that will be used when serving static content (example: /mlflow)"